### PR TITLE
Add translate Vietnam

### PR DESCRIPTION
**Pull request by hngiang**

### CODE REVIEW

**Comment by alexcrichton** [comment]
Thanks for the PR! We typically prefer to keep the translations in this repository, however, right @steveklabnik?

**Comment by steveklabnik** [comment]
Yes, translations reside here; we don't point to other domains.

**Comment by hngiang** [comment]
@alexcrichton @steveklabnik i'll commit version Viet nam right here 👍

**Comment by hngiang** [comment]
@alexcrichton i submited version Vietnam, i change default file `style.css` to `style-vi.css` to support display correct font in Vietnam language, because default font use in css not support display in VN very well. I only remove default font and change to font: `"Helvetica Neue", Helvetica, Arial, sans-serif`. Thanks for reviews.

File: _layouts/fr/basic.html [hunk 1]
```diff
@@ -42,7 +42,8 @@
         <a href="/ko-KR/">한국어</a>,
         <a href="/pt-BR/">Português</a>,
         <a href="/ru-RU/">Русский</a>,
-        <a href="/zh-CN/">简体中文</a>
+        <a href="/zh-CN/">简体中文</a>,
```
Comment (3442853561) [inline _layouts/fr/basic.html]: The letter V is before the letter Z, so please connect the simplified Chinese last. Thank you.

Comment (hngiang) [inline _layouts/fr/basic.html]: Thanks @3442853561 for help me review this commit

**Comment by alexcrichton** [comment]
r? @steveklabnik 

Thanks @hngiang! It looks like some other commits may have snuck in by accident as well?

**Comment by hngiang** [comment]
@alexcrichton it's mean, you see commit of brson in my PR.

**Comment by brson** [comment]
@hngiang GitHub is showing extra commits in your history (from me). I'm not sure why this happened, but it seems like your history is different from what's on master (this happened to one of my PRs too). Can you rebase against master and force push?

I'm not sure what happened, but it's probably my fault. I may have accidentally force pushed to master at some point.

**Comment by hngiang** [comment]
@brson it happen when i pull rebase from master branch rust-lang/www. I will rebase against and force push

**Comment by steveklabnik** [comment]
@hngiang sorry for taking a while to review this!

**Comment by hngiang** [comment]
@steveklabnik you're welcome!

**Pull request merged**